frontend/src/components/InstanceAssignment.vue

```diff
@@ -105,7 +105,16 @@
               />
             </BBTableCell>
             <BBTableCell class="bb-grid-cell">
-              {{ hostPortOfInstanceV1(instance) }}
+              <NTooltip>
```

Comment:
Maybe we can only show the tooltip when there is truncation. 
https://www.naiveui.com/en-US/os-theme/components/ellipsis#basic.vue